package.json

```diff
@@ -50,6 +50,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "jest": "^29.5.0",
     "prettier": "^2.3.1",
+    "timekeeper": "^2.2.0",
```

Comment:
[Info] Needed to freeze time so we can check if a new token has different expiration dates.